test/unittests/mongoBackend/mongoUpdateContext_withOnchangeSubscriptions_test.cpp

```diff
@@ -280,19 +280,23 @@ static void prepareDatabase(bool initializeCache = true) {
                                                      ))
                       );
 
+  LM_M(("Creating 5 entities"));
   connection->insert(ENTITIES_COLL, en1);
   connection->insert(ENTITIES_COLL, en2);
   connection->insert(ENTITIES_COLL, en3);
   connection->insert(ENTITIES_COLL, en4);
   connection->insert(ENTITIES_COLL, en5);
+
+  LM_M(("Creating 5 subscriptions"));
   connection->insert(SUBSCRIBECONTEXT_COLL, sub1);
   connection->insert(SUBSCRIBECONTEXT_COLL, sub2);
   connection->insert(SUBSCRIBECONTEXT_COLL, sub3);
 
   /* Given that preparation including csubs, we have to init cache */
-  if (initializeCache == true)
+  if (useSubCache == true)
   {
     mongoSubCacheInit();
+    mongoSubCacheRefresh();
```

Comment:
Yeah, now the init function does only what a constructor would do.
I am not sure this is the way to go.
Perhaps mongoSubCacheInit should call mongoSubCacheRefresh ...